src/app/store/technical-records/effects/technical-record-service.effects.ts

```diff
@@ -185,6 +194,32 @@ export class TechnicalRecordServiceEffects {
     { dispatch: false }
   );
 
+  updateAxles$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(addAxle, removeAxle),
+      map(() => updateAxlesSuccess())
+    )
+  );
+
+  updateBody$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(updateBody),
+      mergeMap(action =>
+        (this.store.select(selectReferenceDataByResourceKey(ReferenceDataResourceType.PsvMake, action.dtpNumber)) as Observable<PsvMake>).pipe(
+          take(1)
+        )
+      ),
+      map(psvMake => updateBodySuccess({ psvMake }))
+    )
+  );
+
+  updateBrakeForces$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(updateBrakeForces),
+      map(() => updateBrakeForcesSuccess())
+    )
+  );

```

Comment:
I don't really know why this effect and the `updateAxles$` effects are a thing. I assume the `updateBrakeForces` action is caught by a reducer. You can't guarantee the reducer has finished updating state when you dispatch these actions as they are completely independent.

Reply:
The workflow is:
1. Custom section dispatches `updateBrakeForces`
2. That is caught by the reducer and the business logic is carried out
3. `updateBrakeForcesSuccess` is dispatched
4. tech record summary listens to it and refreshes its data.

What is your suggestion?